views/patient/dispensing.html

```diff
@@ -91,12 +89,11 @@
 }
 
 function doneDispensation() {
-  if(sessionStorage.userActivities.search(/Dispensations/ig) > 0) {
-    confirmCancelEntryWithMessage(null, "Are all dispensationsations complete?, continue to patient dashboard?", '/views/patient_dashboard.html?patient_id=' + sessionStorage.patientID, true);
-  }else {
+  if(sessionStorage.userRoles == 'Pharmacist')
+    window.location = "/"
+  else
     window.location = "/views/patient_dashboard.html?patient_id=" + sessionStorage.patientID
-  }
-
+  //nextEncounter(sessionStorage.patientID, 1);
 }
```

Comment:
These were used to verify if all the dispensations were done, why was it removed?

Reply:
we have fixed this too now